dot/sync/chain_sync.go

```diff
@@ -80,6 +79,7 @@ type workHandler interface {
 	handleTick() ([]*worker, error)
 }
 
+//go:generate mockgen -destination=./mock_chain_sync_test.go -package $GOPACKAGE . ChainSync
```

Comment:
nit just add a new line and removed unneeded `./` prefix

```suggestion
//go:generate mockgen -destination=mock_chain_sync_test.go -package $GOPACKAGE . ChainSync

```

Reply:
Done in c2e3d69